src/plugins/intel_npu/src/utils/src/zero/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ target_include_directories(${TARGET_NAME}
 
 target_link_libraries(
   ${TARGET_NAME}
-  PUBLIC openvino::runtime openvino::runtime::dev)
+  PUBLIC openvino::runtime openvino::runtime::dev openvino::npu_al)
```

Comment:
Doesn't look good to ad npu_al to utils.

Reply:
Yes, the deps here is NetworkMeta inside npu_al, If we put this wrapper back to compiler_adapter, do not need to make strange deps here. @pereanub , do you see other regressions? I think we still need other ticket to decouple NetworkMeta with ext wrapper later, can move this back to adapter now to not make blocker to that one